### ISSUE TITLE
Notepad with embedded database: crash when launching quiz

### ISSUE THREAD

**Issue by mjaj** [mvysny/aedict#727]
After switching to the new embedded database and syncing with aedict-online, the app crashes on both my devices when I try to launch a quiz for a notepad category by tapping the quiz button (the three question marks) in the notepad screen. It worked before with the old file format.
It is possible to launch the quiz from the JLPT quiz menu entry by selecting the notepad category, this works well also with the new format.

**Comment by mvysny** [comment]
Thanks for letting me know! I'll look into that. Thanks for being the brave soul ;)

**Comment by mvysny** [comment]
This crash from the Google Play console is probably related:
```
java.lang.UnsupportedOperationException
	at java.util.AbstractList.add(AbstractList.java:404)
	at sk.baka.aedict3.NotepadActivity.quiz(NotepadActivity.java:314)
	at sk.baka.aedict3.NotepadActivity.onOptionsItemSelected(NotepadActivity.java:259)
	at android.app.Activity.onMenuItemSelected(Activity.java:2548)
	at android.support.v4.app.FragmentActivity.onMenuItemSelected(FragmentActivity.java:361)
	at android.support.v7.app.AppCompatActivity.onMenuItemSelected(AppCompatActivity.java:147)
	at android.support.v7.view.WindowCallbackWrapper.onMenuItemSelected(WindowCallbackWrapper.java:100)
	at android.support.v7.app.AppCompatDelegateImplV7.onMenuItemSelected(AppCompatDelegateImplV7.java:620)
	at android.support.v7.view.menu.MenuBuilder.dispatchMenuItemSelected(MenuBuilder.java:811)
	at android.support.v7.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:152)
	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:958)
	at android.support.v7.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:948)
	at android.support.v7.widget.ActionMenuView.invokeItem(ActionMenuView.java:618)
	at android.support.v7.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:139)
	at android.view.View.performClick(View.java:4207)
	at android.view.View$PerformClick.run(View.java:17372)
	at android.os.Handler.handleCallback(Handler.java:725)
	at android.os.Handler.dispatchMessage(Handler.java:92)
	at android.os.Looper.loop(Looper.java:137)
	at android.app.ActivityThread.main(ActivityThread.java:5041)
	at java.lang.reflect.Method.invokeNative(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:511)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:793)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:560)
	at dalvik.system.NativeStart.main(Native Method)
```

**Comment by mvysny** [comment]
Fixed in Aedict 3.39.29
